lightly/cli/embed_cli.py

```diff
@@ -24,7 +26,17 @@
 from lightly.cli._helpers import cpu_count
 
 
-def _embed_cli(cfg, is_cli_call=True):
+def _embed_cli(cfg, is_cli_call=True) -> \
+    Union[
+        Tuple[List[np.ndarray], List[int], List[str]],
```

Comment:
It's not a list of array, it's just an np.ndarray

Reply:
Thanks, good that we add the typehints :)